pure3.md

```diff
@@ -0,0 +1,169 @@
+and therefore will expect other capable ECUs, perhaps the Primary ECU, to perform in-toto
+verification on their behalf. In this model, while every
+ECU may not be verifying the software supply chain of the images it is expected to install,
+verification is still happening on the individual automobile. Essentially, there is no
+difference in the functioning of Partial Verification secondaries and their Full Verification counterparts.
+
+It is important to note that there are some automobiles that have no ECUs capable of providing even a basic level of software supply chain
+security. Therefore, this PURE presents a "stop-gap" option. In this configuration, the repository
+verifies the software supply chain of the image and signs an in-toto _summary link_ with the
+keys used to sign either the top level Targets or delegated signer role. This way, the key used
+to sign the summary link metadata is securely bootstrapped and can be rotated in case of a compromise.
+
+For each target image, the _custom_ field points to the summary link rather than all the in-toto metadata.
+The client must download and verify the Targets metadata for the summary link and, if this passes, download
+and verify the hash and signature on the summary link. Once the authenticity of the summary link is established,
+the client must verify that the hashes in the products field of the summary link match those of the image.
+
+The summary link does offer some evidence that the repository performed in-toto verification
+on behalf of the automobile, but it is **not** a sufficiently strong replacement for actual
+verification on the vehicle. A key issue with the stop-gap option is that it turns the Image repository into a
+single point of failure and therefore can only be trusted as long as the repository remains secure. As such, this model must only be adopted as a stop-gap option until 
+the update system can be transitioned to the full model in which in-toto verification happens on individual vehicles.
+
+
+## Security Analysis
+
+Scudo is an extension for Uptane that adds in-toto's software supply chain security
+properties. Therefore, its threat model is a combination of the threat models of Uptane
+for the delivery of the software artifacts, and in-toto for the development of the image as
+it passes through the software supply chain.
+
+in-toto is a comprehensive framework that secures each step in the software supply chain. Furthermore, it
+plays well with other systems and tools that attempt to secure specific aspects of the
+software supply chain. in-toto can be used to enforce policies pertaining to the flow of artifacts through
+the supply chain and the people or bots authorized to make changes at each step, but does not,
+for example, intrinsically provide mechanisms to store the metadata. The repository can instead
+leverage systems, like Grafeas or Sigstore's Rekor, to store the metadata. Both of these systems already have support
```

Comment:
Next sentence..."Yet, since systems like Grafeas or Sigstore's Rekor already have support for in-toto, the repository can instead leverage these systems to store the metadata."